.rubocop.yml

```diff
@@ -1,5 +1,17 @@
 AllCops:
   TargetRubyVersion: 2.1
+  Include:
+    - 'lib/**/*.rb'
+    - 'test/**/*.rb'
+    - 'Gemfile'
+    - 'Rakefile'
+  Exclude:
+    - 'Appraisals'
+    - '*.gemspec'
+    # TODO: Re-enable these.
+    - 'spec/**/*.rb'
```

Comment:
This is an issue I found while fixing some issues with Rubocop: we aren't delinting RSpec. We should in a future PR.